topics/discordjs/index.md

```diff
@@ -0,0 +1,9 @@
+---	
+aliases: discord.js	
+display_name: Discord.js	
+logo: discordjs.png	
+related: discord	
+short_description: discord.js module to interact with the Discord API	
+topic: discordjs	
+---	
+discord.js is a powerful node.js module that allows you to interact with the Discord API very easily. It takes a much more object-oriented approach than most other JS Discord libraries, making your bot's code significantly tidier and easier to comprehend.
```

Comment:
Overall this topic looks great, thanks for your contribution!!

Would it be possible to reword the description here to be a smidge softer on the other libraries? 😅   This library sounds excellent, but the way this is worded is just a bit more opinionated than we'd prefer to be, I think.

Perhaps
```suggestion
discord.js is a powerful node.js module that allows you to interact with the Discord API very easily. It takes an object-oriented approach, supporting your bot's code to be tidy and easy to comprehend.
```